### PR TITLE
Mobs can no longer ventcrawl while buckled or handcuffed

### LINKED ISSUE #9890: People can ventcrawl when buckled to pipes
Cuff someone
Buckle them to a pipe
Buckled person tries to move
They end up in a pipe
Resist to remove the cuffs
You can now freely ventcrawl anywhere and exit at any vent/scrubber

### PR DESCRIPTION
**Pull request by harmonyn**
- Mobs cannot ventcrawl while buckled to a pipe. Fixes https://github.com/tgstation/-tg-station/issues/9890. It is a bit of a band-aid fix but it is a fix.
- Fixes a related bug: mobs cannot attempt to ventcrawl when they are handcuffed. (It didn't exactly work, as handcuffs are items and it gets stopped at the "do you have items" check, but you shouldn't receive a "You begin crawling into the ventilation system..." message when handcuffed, and now you do not.)

### CODE REVIEW

File: code/ATMOSPHERICS/atmospherics.dm [hunk 1]
```diff
@@ -203,6 +203,9 @@ Pipelines + Other Objects -> Pipe network
 	if(!(direction & initialize_directions)) //cant go this way.
 		return
 
+	if(buckled_mob) // fixes buckle ventcrawl edgecase fuck bug
```
Comment (RemieRichards) [inline code/ATMOSPHERICS/atmospherics.dm]: ... no

```if(buckled_mob == user)```

**Comment by KorPhaeron** [comment]
Couldn't you replace all of this with can_move or something?

**Comment by RemieRichards** [comment]
@KorPhaeron ventcrawling hijacks that var.

**Comment by PKPenguin321** [comment]
this kinda sucks shit for changelings, but whatever

**Comment by bustygoku** [comment]
Can't you go human again to slip out of the cuffs? ^

**Comment by PKPenguin321** [comment]
yeah but it's still gonna be a tight situation, and regardless that doesn't really have much to do with this pr so

**Comment by SconesC** [comment]
Things being able to ventcrawl while cuffed is an obvious bug and I'm glad it's fixed here

**Comment by bgobandit** [comment]
guys

you could never actually ventcrawl when handcuffed, it fails at the "You can't ventcrawl with items!" point

this part is just to fix the misleading message about beginning to crawl into the ventilation system.

(buckle bug did let you though -- the bug existed because being buckled to a pipe bypasses the whole entering-the-pipe process, it thinks you're already in it)

**Comment by Miauw62** [comment]
im sort of sad that this bug is getting fixed because it was really funny

**Pull request merged**